### ISSUE TITLE
[Documentation] : add colors accessibility documentation

### ISSUE THREAD

**Issue by AyalaBu** [Vonage/vivid-3#658]
In the "Getting started" part of Vivid documentation in the "Color Palette" section supposed to be also a "Guidelines - Accessibility" section, but it is missing. 
It needs to be written and reviewed.
